CRM/Upgrade/Incremental/php/FiveTwentyEight.php

```diff
@@ -69,4 +69,30 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
   //   return TRUE;
   // }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_28_alpha1($rev) {
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Populate missing Contact Type name fields', 'populateMissingContactTypeName');
+  }
+
+  public static function populateMissingContactTypeName() {
+    $contactTypes = \Civi\Api4\ContactType::get()
+      ->setCheckPermissions(FALSE)
+      ->execute();
+    foreach ($contactTypes as $contactTypes) {
```

Comment:
Shouldn't that be `($contactTypes as $contactType)` ?  And also $contactType (no 's') in the following lines for id and label?

Reply:
thanks Fixed now